src/resources/formats/pdf/pandoc/latex.template

```diff
@@ -440,13 +445,15 @@ $endfor$
 % get rid of language-specific shorthands (see #6817):
 \let\LanguageShortHands\languageshorthands
 \def\languageshorthands#1{}
+$if(selnolig-langs)$
+\ifLuaTeX
+  \usepackage[$for(selnolig-langs)$$it$$sep$,$endfor$]{selnolig} % disable illegal ligatures
+\fi
+$endif$
 $endif$
 $for(header-includes)$
 $header-includes$
 $endfor$
-\ifLuaTeX
-  \usepackage{selnolig}  % disable illegal ligatures
-\fi
```

Comment:
This should be removed from main template
https://github.com/quarto-dev/quarto-cli/blob/4d6105e73641459723c155c0b7d5f619a34906f5/src/resources/formats/pdf/pandoc/template.tex#L348-L350